src/sidebar/templates/sidebar-tutorial.html

```diff
@@ -41,3 +41,36 @@ <h1 class="sidebar-tutorial__header">How to get started</h1>
     </li>
   </ol>
 </div>
+<div class="sheet sheet--is-theme-clean" ng-if="vm.showSidebarTutorial() && vm.cleanOnboardingThemeEnabled">
+  <i class="close h-icon-close" role="button" title="Close"
+     ng-click="vm.dismiss()"></i>
+  <h1 class="sidebar-tutorial__header sidebar-tutorial__header--is-theme-clean">
+    <i class="h-icon-annotate sidebar-tutorial__header-annotate" h-branding="accentColor"></i>
+    Start annotating
+  </h1>
+  <ol class="sidebar-tutorial__list">
+    <li class="sidebar-tutorial__list-item sidebar-tutorial__list-item--is-theme-clean">
+      <div class="sidebar-tutorial__list-item-content">
+        Select some text to
+          <span class="sidebar-tutorial__list-item-annotate">annotate</span>
+          <svg-icon width="3px" height="10px" class="sidebar-tutorial__list-item-cursor" name="'cursor'"></svg-icon>
```

Comment:
The `width` and `height` properties here are redundant and can be removed since we're now applying the styling in CSS.